src/Resources/config/routing/admin_api.yml

```diff
@@ -0,0 +1,36 @@
+# List of generated routes for better navigation:
+# loevgaard_sylius_brand_admin_api_brand_index
+# loevgaard_sylius_brand_admin_api_brand_create
+# loevgaard_sylius_brand_admin_api_brand_update
+# loevgaard_sylius_brand_admin_api_brand_show
+# loevgaard_sylius_brand_admin_api_brand_delete
```

Comment:
Remove these comments

Reply:
It saves some time for developer to find where route defined.

Lets say, you found some route you want to override `loevgaard_sylius_brand_admin_api_brand_delete` at `bin/console debug:router`. How to find it? There is no chance to find it fast without that comment:
http://prntscr.com/mtbc36

So I would like to keep that comments if you also OK with that.

Reply:
Okay, let's keep them for now :)